lib/TestsRunner.js

```diff
@@ -53,8 +53,8 @@ TestsRunner.prototype.prepareAppToRunTarget = function(target) {
     var me = this;
     return Q.Promise(function(resolve, reject) {
         // if we know external url we can safely use it
-        if (me.server.externalServerUrl) {
-            me.writeMedicLogUrl(me.server.externalServerUrl + ":" + me.server.port);
+        if (me.server.tunneledUrl || me.server.externalServerUrl) {
```

Comment:
I would prefer to have single method to get server access url (including port, etc)
```
server.getConnectionUrl(); // all magic is here
```